drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -290,8 +290,12 @@ function DragAndDropTemplates(configuration) {
             h('div.keyboard-help-dialog', [
                 h('div.modal-window-overlay'),
                 h('div.modal-window', {attributes: {role: 'dialog', 'aria-labelledby': labelledby_id}}, [
+                    h('button.modal-dismiss-button.unbutton', {attributes: {tabindex: 1}}, [
```

Comment:
I left the tabindex here by accident while testing issues with integration tests. Thanks for spotting this, I will remove the attribute.